### DISCUSSION TITLE
Problem with login in on virtualsmarthome

### DISCUSSION THREAD

**Discussion by Zollernalbritter** [AK-Homberger/Alexa-Face-Recognition-with-ESP32CAM#2] [Q&A]
so my problem is that when i wanne try to sign in into the virtual smarthome website i get brought to the amazon login in so i log me there in but then the amazon login website says that they have an problem with there website so is it not anymore working the virtual smarthome website and i can't make the project or is there another way to login?

**Comment by AK-Homberger** [comment]
Hi,
I tested the login right now and it's working for me.
Did you try with a different browser or other security settings in the browser?
Regards,
Andreas

**Reply by Zollernalbritter** [reply]
Thank you very much now its workes the Browser was the Problem i tried it before in Chrome and there it didnt work but now  in Safari it does

Thanks a lot
